website/settings/defaults.py

```diff
@@ -2153,3 +2153,5 @@ def from_node_usage(cls,  usage_bytes, private_limit=None, public_limit=None):
 # path to newrelic.ini config file
 # newrelic is only enabled when DEBUG_MODE is False
 NEWRELIC_INI_PATH = None
+
+EMPTY_REQUEST_INSTITUTIONAL_ACCESS_REQUEST_TEXT = 'User has left custom message field empty'
```

Comment:
Doesn't OSF have a translations strings file that we could put this into instead? We don't use it all over the place, but we have used it. Maybe website/language.py